oneflow/ir/oneflow-extension/kernel_launch/reg_context.cpp

```diff
@@ -95,6 +97,16 @@ RegContext::RegContext(mlir::Operation* op) : op_(op), conf_wrapper_(GetConfWrap
   } else {
     LOG(FATAL) << "Unsupported device tag: " << dev_tag.str();
   }
+  auto op_name = GetOp()->getName().stripDialect().str();
```

Comment:
```suggestion
  const auto op_name = GetOp()->getName().stripDialect().str();
```
在下文中没有发生改变的对象都应该是const的，别的地方也都改一下